packages/guesstimator/lib/distributionMath.js

```diff
@@ -10,3 +10,14 @@ export function distributionUpToIntoLognormal(low, high){
 
     return `lognormal(${mean},${stdev})`
 }
+
+// The left endpoint marks 0.05% on the CDF, the right 0.95%.
+export function distributionUpToIntoNormal(low, high){
+    
+    let mean = math.mean(high, low)
+    let stdev = (high - mean) / 1.645
+    return `normal(${mean},${stdev})`
+
```

Comment:
why the spaces at the top, and especially the bottom? Also, underneath this function. Please delete.

Reply:
Done